src/docs/user.yaml

```diff
@@ -76,7 +76,50 @@ paths:
                                 $ref: '#/components/schemas/ValidationError'
                 500:
                     description: Internal server error
-
+        
+    /api/user/{id}/sub:
+        post:
+            tags:
+                - User
+            summary: Unfollow Users
+            description: Unfollow Users
+            operationId: unFolowUser
+            consumes:
+                - application/json
+            produces:
+                - application/json
+            parameters:
+                - in: path
+                  name: id
+                  description: The id of the user
+                  required: true
+                  type: string
+            responses:
```

Comment:
Same thing here, add the following response code:
```
Can you please add thıs response code, since your endpoint need an authenticated user:

403:
  description: This status is returned when the user is not authorized to perform the operation.
```